### PR TITLE
https://issues.redhat.com/browse/ACM-9296 Nutanix prereq

### PR DESCRIPTION
**Pull request by oafischer**
2.9 only

### CODE REVIEW

**Comment by jelkosz** [comment]
lgtm

File: clusters/cluster_lifecycle/create_cluster_cli.adoc [hunk 1]
```diff
@@ -28,6 +28,21 @@ find clusterImageSets/fast -type d -exec oc apply -f {} \; 2> /dev/null
 
 Select the {ocp} release images when you create a cluster.
 
+*Note:* If you use the Nutanix platform, make sure that the `ClusterImageSet` that you want to use for the Nutanix deployment has a `x86_64` `releaseImage` and that the `visible` annotation has the `'true'` value. See the following example:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_cluster_cli.adoc]:
```suggestion
*Note:* If you use the Nutanix platform, be sure to use `x86_64` architecture for the `releaseImage` in the `ClusterImageSet` resource and set the `visible` annotation value to `true`. See the following example:
```

File: clusters/cluster_lifecycle/create_cluster_cli.adoc [hunk 2]
```diff
@@ -28,6 +28,21 @@ find clusterImageSets/fast -type d -exec oc apply -f {} \; 2> /dev/null
...
+
+[source,YAML]
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_cluster_cli.adoc]: all instances throughout the doc uses lowercase. Link to ASCII doc source block: https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-blocks/ 
```suggestion
[source,yaml]
```

File: clusters/cluster_lifecycle/create_cluster_cli.adoc [hunk 3]
```diff
@@ -28,6 +28,21 @@ find clusterImageSets/fast -type d -exec oc apply -f {} \; 2> /dev/null
...
+----
+apiVersion: hive.openshift.io/v1
+kind: ClusterImageSet
+metadata:
+  labels:
+    channel: stable
+    visible: 'true'
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_cluster_cli.adoc:40]: Will you verify if the single quotes are required? I believe this is another clean up that we need to do (Remove quotation marks out of YAML samples)
```suggestion
    visible: true
```

**Review by dockerymick — CHANGES_REQUESTED** [review]
Thanks for making the updates! I made some suggestions

File: clusters/cluster_lifecycle/create_cluster_cli.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/create_cluster_cli.adoc:40]: @jelkosz ?

Comment (jelkosz) [reply clusters/cluster_lifecycle/create_cluster_cli.adoc:40]: The apostrophe has to be there, the type is string here.

Comment (oafischer) [reply clusters/cluster_lifecycle/create_cluster_cli.adoc:40]: Thanks for confirming! And thanks for reviewing @dockerymick, let me know if it looks good now.

Comment (dockerymick) [reply clusters/cluster_lifecycle/create_cluster_cli.adoc:40]: Thanks Oliver! Taking a look now

File: clusters/cluster_lifecycle/create_cluster_cli.adoc [hunk 4]
```diff
@@ -28,6 +28,21 @@ find clusterImageSets/fast -type d -exec oc apply -f {} \; 2> /dev/null
 
 Select the {ocp} release images when you create a cluster.
 
+*Note:* If you use the Nutanix platform, be sure to use `x86_64` architecture for the `releaseImage` in the `ClusterImageSet` resource and set the `visible` annotation value to `true`. See the following example:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_cluster_cli.adoc]: Adding single quote back to the value  @oafischer 
```suggestion
*Note:* If you use the Nutanix platform, be sure to use `x86_64` architecture for the `releaseImage` in the `ClusterImageSet` resource and set the `visible` annotation value to `'true'`. See the following example:
```

**Review by dockerymick — APPROVED** [review]
Added the single quotes back after feedback from @jelkosz 
Other than that, this looks good to me !

**Pull request merged**